app/src/test/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityServiceTest.kt

```diff
@@ -489,3 +593,9 @@ private val INVALID_LOGIN = GetTokenResponseJson.Invalid(
         errorMessage = "123",
     ),
 )
+
+private val SEND_VERIFICATION_EMAIL_REQUEST_JSON = SendVerificationEmailRequestJson(
```

Comment:
The `_JSON` suffix is reserved for a Json string, this would just be `SEND_VERIFICATION_EMAIL_REQUEST`